server/swap/swap.go

```diff
@@ -1909,17 +1981,19 @@ func (s *Swapper) processRedeem(msg *msgjson.Message, params *msgjson.Redeem, st
 	// Expire function to unregister the outstanding request.
 	expireFunc := func() {
 		s.rmLiveAckers(notification.ID)
-		log.Infof("Timeout waiting for redeem ack request from user %v (%s).",
-			ack.user, makerTaker(ack.isMaker))
+		log.Infof("Timeout waiting for 'redemption' request from user %v (%s) for match %v",
+			ack.user, makerTaker(ack.isMaker), matchID)
 	}
 
 	// Send the ack request.
 
-	// The counterparty does not need to actually locate the redemption on txn,
-	// so use the default request timeout.
+	// The counterparty does not need to actually locate the redemption txn, but
+	// they do need to author and broadcast their own redeem, so allow the
+	// request to wait for a response up until the inaction penalty.
 	s.authMgr.RequestWhenConnected(ack.user, notification, func(_ comms.Link, resp *msgjson.Message) {
 		s.processAck(resp, ack) // resp.ID == notification.ID
-	}, auth.DefaultRequestTimeout, auth.DefaultConnectTimeout, expireFunc)
+	}, s.bTimeout, auth.DefaultConnectTimeout, expireFunc)
+	// maybe: s.bTimeout-time.Since(redeemTime) or time.Until(redeemTime.Add(s.bTimeout))
```

Comment:
This pertains to the issue resolved by the final commit, where we saw a maker with simultaneous 20+ matches that were processed in series instead of concurrently.  The timeout on the response was too short.  Even though the client is not doing them concurrently and faster, I'm not sure I see a reason to timeout on the response before inaction time.

However, a different RequestWhenConnected with a single *deadline* instead of two *timeouts* for response and connect would be more appropriate and less confusing.